### ISSUE TITLE
What is the shift operation in the SUNRGBD data set for?

### ISSUE THREAD

**Issue by hyx1999** [laughtervv/DepthAwareCNN#34]
`depth = np.bitwise_or(np.right_shift(depth,3),np.left_shift(depth,16-3))`
I am currently reproducing this paper, so I am studying the author's implementation method. I can't understand this code in sunrgbd_dataset.py. Could someone please give me some help?

**Comment by jiunyen-ching** [comment]
`np.right_shift` and `np.left_shift` are referring to the bitshiting operations. The depth image was bitshifted to provide better visualization. However, these pixel values are not the **_real_** depth values that we can directly use for calculation operations. So, when loading the data, bitshifting needs to be reversed via 
`depth = np.bitwise_or(np.right_shift(depth,3),np.left_shift(depth,16-3))`

For example, consider this depth image without any bitshifting modification (Image A)
![image](https://user-images.githubusercontent.com/25114361/158779539-2b520948-db00-4aa6-92f2-6a7d1bf035de.png)
versus a bitshifted depth image (Image B)
![image](https://user-images.githubusercontent.com/25114361/158779686-aac1d7d4-8444-4cf4-885f-3eca2396df47.png)

Instantly we would be able to make sense of the shapes and depth differences between the objects in Image B compared to A.
 
In summary, the two operations involved are
A -> B : depth_b = np.bitwise_or(np.right_shift(depth_a, 16-3), np.left_shift(depth_a,3))
B -> A : depth_a = np.bitwise_or(np.right_shift(depth_b,3), np.left_shift(depth_b,16-3))
